tests/main/install-sideload/task.yaml

```diff
@@ -1,5 +1,7 @@
 summary: Checks for snap sideload install
 
+systems: [-debian-*]
```

Comment:
Please always indicate why something is disabled on any given system. Otherwise it is just a bug waiting to be discovered.